### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by gabyse1**
## Changes to the project
- Create the *crud.js* file to export add, remove and edit tasks.
- Implement a function for adding a new task. The property *complete* is set in *false* by default, and the tasks' *indexes* are created according to the array's length. 
- Implement a function for deleting tasks. This updates the *indexes* of the remaining tasks.
- Implement a function for editing task descriptions.
- All changes to the *To Do List* are saved in local storage.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const loadLocalStorage = (arr) => {
+  const localToDoList = JSON.parse(localStorage.getItem('toDoList'));
+  if (localToDoList) arr = localToDoList;
+  return arr;
+};
+
+const addTask = (arr, task) => {
+  arr.push({ id: arr.length + 1, description: task, completed: false });
+  saveLocalStorage(arr);
+};
```
Comment (TedLivist) [inline src/crud.js:14]: - [ ] Take a look at the image below. Your tasks are missing the `index` property as stated in the requirements. Kindly add this property.

  gaby-index

File: src/crud.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (TedLivist) [inline src/crud.js:15]: - [ ] You are missing the functionality for deleting a single task. Kindly add this functionality. And also add some kind of icon (or button, or element) to each task that triggers this function when clicked. Remember to update the `index` property for remaining items after deleting a task.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gabyse1,

Good job so far!
✔️ Good documentation
✔️ Clean code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @gabyse1,

Your implementation of the functionalities is awesome 💯.
You have also done well addressing previous review concerns 👍🏼.

Your project is now complete!  Consider merging this branch :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**